src/main/java/com/google/sps/servlets/utils/UserComment.java

```diff
@@ -35,15 +38,66 @@ public UserComment(CommentThread commentThread) {
     this.publishDate = topLevelComment.getSnippet().getPublishedAt();
   }
 
-  public String getCommentId() {
-    return commentId;
+  /** This is a easy way to construct a new userComment object for testing purpose. */
+  public UserComment(
+      String commentId, String commentMsg, DateTime publishDate, double score, double magnitude) {
+    this.commentId = commentId;
+    this.commentMsg = commentMsg;
+    this.publishDate = publishDate;
+    this.magnitude = magnitude;
+    this.score = score;
```

Comment:
Change these to maybe magnitudeScore and sentimentScore? Just so it's clear

Reply:
I agree with this